src/_helpers/theme.js

```diff
@@ -6,6 +6,7 @@ const makePxValue = (value) => {
   return values.map((v) => (typeof v === 'string' ? v : `${v * spacings.unit}px`)).join(' ');
 };
 
+// given a color and shade, returns a colorKey.
 const getColorKey = (color, shade) => `${color}.${shade}`;
```

Comment:
Why do we need this? Can't we just directly use the color and shade in the component?

Reply:
Yeah, discussed this and we are removing it.